def.bzl

```diff
@@ -34,6 +36,7 @@ pkg_tar2rpm = rule(
         "preun": attr.string(),
         "postun": attr.string(),
         "requires": attr.string_list(),
+        "build_time": attr.string(),
```

Comment:
Maybe just add this with a default value of 0, so there's no need to check for build_time with args.add

Reply:
Also, it may be worth adding a doc string with the expected format of the build_time.

Reply:
That's a good idea, but once again, because the zero value is not exactly the zero time, I'd rather not pass the value at all.